HW5/SikorskaViktoriia/lab.5.3.py

```diff
@@ -0,0 +1,5 @@
+n = int(input("Enter number -> "))
```

Comment:
good job
look at this version of code
input_number = int(input("Input natural number: "))

fact = 1

if input_number == 1 or input_number == 0:
    print("Factorial {}! =".format(input_number), fact)
elif input_number < 0:
    print("Factorial non positive number don't exist")
else:
    for item in range(1,input_number + 1):
        fact *= item
    print("Factorial {}! =".format(input_number), fact)